chaincode/utils.go

```diff
@@ -118,3 +120,46 @@ func GetTxCreator(stub shim.ChaincodeStubInterface) (string, error) {
 
 	return sID.GetMspid(), nil
 }
+
+// LowerFirst returns the input string with the first letter lowercased
+func LowerFirst(s string) string {
+	if s == "" {
+		return ""
+	}
+	r, n := utf8.DecodeRuneInString(s)
+	return string(unicode.ToLower(r)) + s[n:]
```

Comment:
I'm not a fan to use a string to rune to string conversion but I don't have strong arguments to sustain such statement. But just to discuss it, what do you think of something like:
```go
t := strings.SplitN(s,"",2)
return strings.ToLower(t[0]) + t[1]
```

Reply:
+1 that makes sense!

Reply:
Not that much now that I think about it.... It will panic for a one character string!

Reply:
```go
t := strings.SplitN(s,"",2)
t[0] = strings.ToLower(t[0])
return strings.Join(t, "")
```
Maybe?